src/index.ts

```diff
@@ -32,7 +32,10 @@ async function start() {
 		false
 	)
 	manager.install(managerModule(manager), false)
-	manager.installMultiple(await ModuleManager.directory('externals'), true)
+	manager.installMultiple(
+		await ModuleManager.directory(join(process.cwd(), 'externals')),
```

Comment:
How about making the behavior consistent with the module manager?

```suggestion
		await ModuleManager.directory(__dirname, '..', 'externals')),
```

Reply:
I will update if this works fine